pages/in/widget/iscn/index.vue

```diff
@@ -297,8 +305,27 @@ export default {
       if (!window) return null;
       return window.opener;
     },
-    ipfsURL() {
-      return `https://ipfs.io/ipfs/${this.fingerprint}`;
+    fingerprintURLs() {
+      const fingerprintList = [];
+      for (let i = 0; i < this.fingerprint.length; i += 1) {
+        const fingerprintParts = this.fingerprint[i].split('://');
+        let url = '';
+        switch (fingerprintParts[0]) {
+          case 'ipfs':
+            url = `https://ipfs.io/ipfs/${fingerprintParts[1]}`;
+            break;
+          case 'ar':
+            url = `https://arweave.net/${fingerprintParts[1]}`;
+            break;
+          case 'hash':
+            url = `https://mainnet-node.like.co/txs/${fingerprintParts[1]}`;
+            break;
```

Comment:
I dont think we have a `hash` type that is a transaction hash, we do use file sha256 hash but it doesn't have a url
Please remove this case